### ISSUE TITLE
Import Your Own Data

### ISSUE THREAD

**Issue by dsambor10** [holgerteichgraeber/TimeSeriesClustering.jl#130]
Hi, 
I tried importing my data via your instructions and when I run these lines:

    my_path=joinpath(homedir(),"Documents","tutorial","TimeClustInput.csv")
    your_data_1=load_timeseries_data(my_path; region="none", T=24)

I get this error:
    LoadError: MethodError: objects of type Bool are not callable

Sorry for the basic question!
Dan

**Comment by YoungFaithful** [comment]
Hi @dsambor10, I suppose your `TimeClustInput.csv` contains true and false, is that correct?

**Comment by dsambor10** [comment]
That's the confusing aspect, it does not. Is there a proper timestamp format that is required? Here's a snapshot of my data: 
![image](https://user-images.githubusercontent.com/66494591/84576382-31fc2180-ad69-11ea-8dcf-0acec89abe6c.png)

**Comment by YoungFaithful** [comment]
Your timestamp column looks all right. However you need to add a column called `Year`. (That is due to the fact, that the timestamp is not really interpretated)
https://holgerteichgraeber.github.io/TimeSeriesClustering.jl/stable/load_data/#Load-your-own-data-1

**Comment by dsambor10** [comment]
Thanks for clarifying. I discovered the issue was that I had not specified in loading time series that "years=[2019]" because the doc said that's optional to designate and I thought it would read the year, but it's required to have I guess. It all works now. 
PS Is there any means of exporting the .clust_data (e.g. to a csv) once it's been clustered?

**Comment by holgerteichgraeber** [comment]
I don't think we have a function for that. Should be pretty simple to implement though: 
```
for (k,v) in X.clust_data.data  
  name = "$k"
  CSV.write(name*".csv", DataFrame(v))
end
```

clust_data is of type ClustData, and .data is a dictionary, see https://github.com/holgerteichgraeber/TimeSeriesClustering.jl/blob/master/src/utils/datastructs.jl

You may need to play with the settings of DataFrame and CSV.write, but this should give a general idea.

This will write one csv for each data type.

**Comment by AkshayaTR** [comment]
Hi,
I also happen to be facing an issue with importing my data.
The content in the file has the correct format as needed, but I get the following error:

MethodError: no method matching depwarn(::String, ::Symbol; force=true)
Closest candidates are:
depwarn(::Any, ::Any) at deprecated.jl:79 got unsupported keyword argument "force"

Any idea on what's wrong would be very helpful.

Thanks,
Akshaya

**Comment by dsambor10** [comment]
I'm getting the same error that I and others have gotten in the past "objects of type Bool are not callable" (as per screenshot below) when I try to load my time series data (screenshot) below. It was working before but now appears to be an issue again. I've designated a year column and all my paths are correct. I'm loading my data with: 

ts_input_data = load_timeseries_data("/home/dsambor/DanDocs/KLRS/Clust"; T=24,years=[2021])

![image](https://user-images.githubusercontent.com/66494591/136268724-3b7d26d3-5013-456f-a10e-0706413ca5cc.png)

![image](https://user-images.githubusercontent.com/66494591/136268668-01d16ea1-2a79-4a06-bbf9-0f6b506cbc55.png)
